pom.xml

```diff
@@ -90,6 +90,7 @@
         <json.smart.version>2.4.7</json.smart.version>
         <thrift.version>0.13.0</thrift.version>
         <tomcat.version>8.5.65</tomcat.version>
+        <io.confluent.kafka-connect-avro-data.version>${confluent.version.range}</io.confluent.kafka-connect-avro-data.version>
```

Comment:
I think this variable, if kept, should end with .range

Reply:
This is a similar pattern we use even in common https://github.com/confluentinc/common/blob/24293ac396d07b36fb125fa3c4e037565782c669/pom.xml#L156 . Picked it up from [nanoversioning guide](https://confluentinc.atlassian.net/wiki/spaces/TOOLS/pages/1298340829/Nano+Versioning+User+Guide#Declare-the-Version-Range)